docs/tutorial/using-checkbox/installing_checkbox.rst

```diff
@@ -0,0 +1,56 @@
+===================
+Installing checkbox
+===================
+
+In order to install checkbox you are going to need two parts, a ``runtime``
+and a ``frontend``.
+
+First we need to install the ``runtime``::
+
+   $ sudo snap install checkbox22
+   Ensure prerequisites for "checkbox22" are available
+   Fetch and check assertions for snap "checkbox22"
+   Mount snap "checkbox22"
+   Mount snap "checkbox22"
+   Copy snap "checkbox22" data
+   Run configure hook of "checkbox22" snap if present
+   checkbox22 X.Y.Z from Canonical Certification Team (ce-certification-qa) installed
+
+Now that we have the ``checkbox22`` runtime, we need a ``frontend``, to install it
+run the following::
+
+  $ sudo snap install checkbox --channel uc22
```

Comment:
Most people are using Ubuntu desktop, so they should be using the classic version of that snap, such as

```
snap install checkbox --channel=22.04/stable --classic
```

I'm not sure if we should introduce an explanation of the different channels and tracks here, but it's true that the output of `snap info checkbox` is scary.

Reply:
I thought about this (I even tried to create a sphinx table). I think we must pick one size fits all and defer to reference (as I did in the note) for more info. A new user does not know, and does not need to know, what the difference between uc22 and 22.04 is. Also, that is quite unclear to me as well

Reply:
UC22 is the version for Ubuntu Core 22. @yphus can give you more information about the difference, but basically:

- it's not possible to install a classic snap on Ubuntu Core
- in our case, classic snaps behave closer to the Debian packaged version on desktops
- classic snaps have access to any commands/packages installed on the device, not just the stuff packaged into their own snap

A user will be installing and testing Checkbox on a classic desktop environment, so it's better to use the classic version of the `checkbox` snap. It's still important to let the user know they have to install the runtime and frontend that matches their device's Ubuntu version. Maybe you can just use Ubuntu 22.04 (`checkbox22` runtime snap + `checkbox` snap from `22.04/stable` channel) and 20.04 (`checkbox20` runtime snap + `checkbox` snap from `20.04/stable` channel) as two examples, so that the user knows there are different versions.

Reply:
It still appears to me as `$ sudo snap install checkbox --channel uc22`. Shouldn't it be changed to `snap install checkbox --channel=22.04/stable --classic` after the discussion above? Let me know if I misunderstood it.
Anyway, I follow the steps here, installed frontend with uc22 on my 22.04 Desktop, and failed the next step `checkbox.checkbox-cli` due to lacking of permission to create folder `/var/tmp/checkbox-ng`.

```
nancy@test2:~$ sudo snap install checkbox22
[sudo] password for nancy: 
checkbox22 2.7 from Canonical Certification Team (ce-certification-qa) installed
nancy@test2:~$ sudo snap install checkbox --channel uc22
checkbox (uc22/stable) 2.7 from Canonical Certification Team (ce-certification-qa) installed
nancy@test2:~$ checkbox.checkbox-cli 
/snap/checkbox/2484/checkbox-runtime/wrapper_common: line 38: /usr/bin/findmnt: Permission denied
/snap/checkbox/2484/checkbox-runtime/wrapper_common: line 38: /usr/bin/findmnt: Permission denied
=========================[ Resume Incomplete Session ]==========================
There are 0 incomplete sessions that might be resumed
Preparing...
ERROR:plainbox.bug:Undeclared exception PermissionError raised from start_new_session
Traceback (most recent call last):
  File "/snap/checkbox/2484/checkbox-runtime/bin/checkbox-cli", line 8, in <module>
    sys.exit(main())
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/checkbox_ng/launcher/checkbox_cli.py", line 140, in main
    subcmd.invoked(ctx)
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/checkbox_ng/launcher/subcommands.py", line 205, in invoked
    self._start_new_session()
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/checkbox_ng/launcher/subcommands.py", line 350, in _start_new_session
    self.ctx.sa.start_new_session(title, UnifiedRunner, runner_kwargs)
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/plainbox/impl/decorators.py", line 142, in wrapper
    raise exc
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/plainbox/impl/decorators.py", line 136, in wrapper
    return func(*args, **kwargs)
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/plainbox/impl/session/assistant.py", line 483, in start_new_session
    self._manager = SessionManager.create(prefix=title + '-')
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/plainbox/impl/session/manager.py", line 166, in create
    storage = SessionStorage.create(prefix)
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/plainbox/impl/session/storage.py", line 224, in create
    WellKnownDirsHelper.populate_base()
  File "/snap/checkbox/2484/checkbox-runtime/lib/python3.10/site-packages/plainbox/impl/session/storage.py", line 64, in populate_base
    os.makedirs(dirname)
  File "/snap/checkbox/2484/checkbox-runtime/usr/lib/python3.10/os.py", line 225, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/var/tmp/checkbox-ng'
```

Reply:
+1, fixed all to classic snaps